control-plane/p4RuntimeSerializer.h

```diff
@@ -35,12 +35,13 @@ enum class P4RuntimeFormat {
   TEXT
 };
 
-void serializeP4Runtime(std::ostream* destination,
+void serializeP4Runtime(std::ostream* p4InfoDestination,
                         const IR::P4Program* program,
                         const IR::ToplevelBlock* evaluatedProgram,
                         ReferenceMap* refMap,
                         TypeMap* typeMap,
-                        P4RuntimeFormat format = P4RuntimeFormat::BINARY);
+                        P4RuntimeFormat format = P4RuntimeFormat::BINARY,
+                        std::ostream* p4RuntimeEntriesDestination = nullptr);
```

Comment:
Once #913 gets merged, you will be able to implement this much more cleanly as a separate method on P4RuntimeAPI.

Reply:
sounds good